apps/dashboard/src/components/workflow-editor/test-workflow/snippet-editor.tsx

```diff
@@ -1,40 +1,5 @@
-import { Editor } from '@/components/primitives/editor';
-import { LanguageName, loadLanguage } from '@uiw/codemirror-extensions-langs';
-import { EditorView } from '@uiw/react-codemirror';
-import { useMemo } from 'react';
-import type { SnippetLanguage } from './types';
+import { CodeBlock, Language } from '../../primitives/code-block';
 
-const basicSetup = { lineNumbers: true };
-
-export const SnippetEditor = ({
-  language,
-  value,
-  readOnly = false,
-}: {
-  language: SnippetLanguage;
-  value: string;
-  readOnly?: boolean;
-}) => {
-  const editorLanguage: LanguageName = language === 'framework' ? 'typescript' : language;
-
-  const extensions = useMemo(() => {
-    const res = [EditorView.lineWrapping];
-    const langExtension = loadLanguage(editorLanguage)?.extension;
-    if (langExtension) {
-      res.push(langExtension);
-    }
-    return res;
-  }, [editorLanguage]);
-
-  return (
-    <Editor
-      readOnly={readOnly}
-      lang={editorLanguage}
-      className="h-full"
-      value={value}
-      extensions={extensions}
-      basicSetup={basicSetup}
-      multiline
-    />
-  );
+export const SnippetEditor = ({ language, value }: { language: Language; value: string }) => {
+  return <CodeBlock theme="light" className="h-full overflow-auto" language={language} code={value} />;
```

Comment:
I've updated it to use the code block reusable component instead of the current code mirror implementation to consolidate look and feel across all the snippets in the app